src/common/components/DataView.module.scss

```diff
@@ -0,0 +1,30 @@
+.dataview_no_data {
+  font-style: italic;
+  padding: 20px;
+  text-align: center;
+}
+
+.dataview_row_outer {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  margin-bottom: 5px;
+  padding-left: 0.5rem;
+  padding-top: 0.25rem;
+}
+
+.dataview_row_inner {
+  margin-left: 2rem;
+}
+
+.dataview a,
+.dataview a:visited {
+  color: #000;
+  text-decoration: none;
+}
+
+.dataview_row_type {
+  color: rgba(0, 0, 0, 0.8);
```

Comment:
For colors in this file, could they be references to colors in `src/common/colors.scss` (or could they be changed to be one of those colors?).